### ISSUE TITLE
### <span aria-hidden="true">❌</span> Deploy Preview for *infamousfreight* failed.

### ISSUE THREAD

**Issue by MrMiless44** [MrMiless44/Infamous-freight-enterprises#352]
### <span aria-hidden="true">❌</span> Deploy Preview for *infamousfreight* failed.

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 79a6243a95d3583e5b4feb1beb5bc771e80be83e |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/projects/infamousfreight/deploys/6973475ceb834d000807226e |

_Originally posted by @netlify[bot] in https://github.com/MrMiless44/Infamous-freight-enterprises/pull/350#issuecomment-3789452798_
